manuscript/manuscript.Rmd

```diff
@@ -602,19 +626,44 @@ One recently proposed framework to overcome challenges with explicit reward desi
 E.g. rarely does an action in conservation lead to immediate rewards; instead, benefits are typically observed after actions have been performed.
 Sparse reward structures are challenging for DRL agents, because of the credit assignment problem [cite].
 To solve sparse reward environments, many researchers have employed customized algorithms to achieve better performance than standard DRL methods [cite Montezuma].
 For conservation, we anticipate that there will likely be a similar pattern, wherein bespoke RL methods will attain better performance than general methods.
 
--   wild west of AI?
-    -can separate RL from having gyms implemented -\> how do you compare different models, -mention that customization is probably the way to go, baselines only go so far
+<!-- But never fear, hard problems have been solved before!-->
+Deep RL is still a very young field, where despite several landmark successes, potential far outstrips practice.
+Recent advances in the field have proven the potential of the approach to solve complex problems [@alphaGo2016; @alphaGoZero; @alphazero; @DQN], but typically leveraging large teams with decades of experience in ML and millions of dollars worth of computing power [@alphaGoZero].
+Successes have so far been concentrated in applications to games and robotics, not scientific and policy domains, though this is beginning to change [@protein_folding].
+Leading developers of new algorithms for deep RL benchmark the performance of their algorithms against such problems in part because simulations of games and robotic movement are readily available through open source frameworks designed to interface with RL algorithms [@gym], and performance can easily be compared against alternatives on public leaderboards [@gym].
+Iterative improvements to well-posed public challenges have proven immensely effective in the computer science community in tackling difficult problems, which allow many teams with diverse expertise not only to compete but to learn from each other [@netflix_prize; @imagenet].
+Advances in both computational power and algorithmic efficiency continue to reduce costs at a dramatic pace, significantly faster than Moore's law [@openai_costs].
+By working to develop similarly well-posed challenges as clear benchmarks, ecology and environmental science researchers may both be better able to replicate that collaborative, iterative success in cracking hard problems.
+Such benchmarks may further improve collaboration across disciplines as well, attracting the attention of AI expertise to these key issues facing our planet.
+If RL opens the door to greater role for artificial intelligence and engagement of technology firms in the machinery that determines environmental policy, this raises new concerns as well as new opportunities.  
+
+
+<!-- transition from big promise to perils -- Millie's section on risks here -->
```

Comment:
tbd sry I'm taking a while on this! I really like the set up here -- will be sure to engage the private/public axis of the issue now!

Reply:
@cboettig I added some text in the 'millie-edits' branch on this (can paste here if it easier to keep discussion edits all on one branch?)